lib/use-cases/send-referral-response-email.js

```diff
@@ -15,7 +15,7 @@ export default class SendReferralResponseEmail {
       name: referral.name,
       referrerName: referral.referrerName,
       serviceName: referral.serviceName,
-      referenceCode: referral.referenceNumber || '',
+      referenceNumber: referral.referenceNumber || '',
```

Comment:
This will need changing in the template too. I checked the Rejected template which shows referenceCode.

Reply:
Ah, yes. We're going to be deleting those templates entirely as they're duplicates of the current ones (that are without a variable yet). We'll update the GovUK Notify templates once the wording of where we have to put in that reference number becomes clearer.